src/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -142,4 +142,18 @@
     <protected>false</protected>
     <shortDescription>Abandon Request</shortDescription>
   </labels>
+  <labels>
+    <fullName>cookbook_Next</fullName>
+    <value>Next</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Next</shortDescription>
+  </labels>
+  <labels>
+    <fullName>cookbook_Back</fullName>
```

Comment:
I would not combine the terms `Back` and `Next`. Usually, we see either "Back/Forward" or "Previous/Next". In a wizard-like interface, "Previous/Next" are much more common. So I'd suggest to use "Previous" instead of "Back".